iothub/device/src/WritablePropertyResponse.cs

```diff
@@ -2,18 +2,29 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
-namespace PnpHelpers
+namespace Microsoft.Azure.Devices.Client
 {
     /// <summary>
-    /// The payload for a property update response.
+    ///
     /// </summary>
     public class WritablePropertyResponse
     {
+        private readonly IPayloadConvention _payloadConvention;
+
         /// <summary>
-        /// Empty constructor.
+        /// Convenience constructor for specifying only the property value.
         /// </summary>
-        public WritablePropertyResponse() { }
+        /// <param name="propertyValue">The unserialized property value.</param>
+        /// <param name="payloadConvention"></param>
+        public WritablePropertyResponse(object propertyValue, IPayloadConvention payloadConvention = default)
```

Comment:
this constructor isn't required (value, ackCode and ackVersion are required)

Reply:
Done